src/results/results.jl

```diff
@@ -35,8 +35,8 @@ Create a dictionary of results with string keys for each Scenario structure mode
 function reopt_results(m::JuMP.AbstractModel, p::REoptInputs; _n="")
 	tstart = time()
     d = Dict{String, Any}()
-    for b in p.s.storage.types
-        if p.s.storage.max_kw[b] > 0
+    for b in p.s.storage.types.all
+        if p.s.storage.attr[b].max_kw > 0 && p.s.storage.attr[b].max_kwh > 0
```

Comment:
This ostensibly covers all storage results, but I'll add a TODO to check what other storage results that are unique to thermal and/or electric storage types.